lib/galaxy/datatypes/binary.py

```diff
@@ -776,7 +768,64 @@ def display_peek(self, dataset):
         try:
             return dataset.peek
         except Exception:
-            return "Binary GROMACS trr file (%s)" % (nice_size(dataset.get_size()))
+            return "Binary GROMACS %s trajectory file (%s)" % (self._file_ext, nice_size(dataset.get_size()))
+
+
+class Trr(GmxBinary):
+    """
+    Class describing an trr file from the GROMACS suite
+
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('md.trr')
+    >>> Trr().sniff(fname)
+    True
+    >>> fname = get_test_fname('interval.interval')
+    >>> Trr().sniff(fname)
+    False
+    """
+    file_ext = "trr"
```

Comment:
So far instance here you put `magic_number = 1993` and then you don't need any `def __init__()`